environment_maple_macos.yml

```diff
@@ -0,0 +1,15 @@
+name: maple_py39
+channels:
+  - defaults
+  - apple
+dependencies:
+  - python=3.9
```

Comment:
same comment as above, please set to 3.10

Reply:
updated.